<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bilancaisse</class>
 <widget class="QDialog" name="bilancaisse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_bilan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>861</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_statistique">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RechercherDe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher De:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RechercherA">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ã </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_BilanDatedebut">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_BilanDatefin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
